core/lib/types/src/block.rs

```diff
@@ -84,6 +84,16 @@ pub struct MiniblockHeader {
     pub virtual_blocks: u32,
 }
 
+/// Consensus-related L2 block (= miniblock) fields.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ConsensusBlockFields {
+    /// Hash of the previous consensus block.
+    pub prev_block_hash: H256,
+    /// Protobuf serialization of quorum certificate for the block.
+    pub commit_qc_bytes: Bytes,
```

Comment:
can we make it a dedicated type? Either a wrapper of bytes, or import CommitQC struct from consensus crate directly?